Dockerfile-development

```diff
@@ -1,37 +1,37 @@
 ARG BUILDER_IMAGE
 FROM ${BUILDER_IMAGE}
 LABEL maintainer="dortiz@redhat.com"
+ENV GOSU_VERSION 1.10
```

Comment:
Why? I think this whole Dockerfile can be removed after the cleanup in https://github.com/3scale/apicast/pull/751